eng/jobs/linux-build.yml

```diff
@@ -22,9 +22,11 @@ jobs:
   - job: ${{ parameters.displayName }}
     displayName: ${{ parameters.displayName }}
     pool:
-      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      ${{ if eq(parameters.displayName,'Build_Linux_x64_FreeBSD') }}:
+        name: dnceng-freebsd-internal
+      ${{ if and(eq(variables['System.TeamProject'], 'public'), ne(parameters.displayName,'Build_Linux_x64_FreeBSD')) }}:
         name: dnceng-linux-external-temp
-      ${{ if ne(variables['System.TeamProject'], 'public') }}:
+      ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(parameters.displayName,'Build_Linux_x64_FreeBSD')) }}:
```

Comment:
Can this be nested so `ne(parameters.displayName,'Build_Linux_x64_FreeBSD')` is only written out once?

Reply:
I couldn't find an example of nested if and wasn't sure if it is supported. Will look up and update.